src/sage/schemes/hyperelliptic_curves/hypellfrob.pyx

```diff
@@ -175,24 +162,17 @@ def hypellfrob(p, N, Q):
     INPUT:
 
     - ``p`` -- a prime
-    - ``Q`` -- a monic polynomial in `\ZZ[x]` of odd degree.
-      Must have no multiple roots mod `p`.
-    - ``N`` -- precision parameter; the output matrix will be correct modulo `p^N`.
-
-    PRECONDITIONS:
+    - ``Q`` -- a monic polynomial in `\ZZ[x]` of odd degree; must have no
+      multiple roots mod `p`.
+    - ``N`` -- precision parameter; the output matrix will be correct modulo `p^N`
 
-    Must have `p > (2g+1)(2N-1)`, where `g = (\deg(Q)-1)/2` is the genus
-    of the curve.
+    The prime `p` should satisfy `p > (2g+1)(2N-1)`, where `g =
+    \frac{\deg(Q)-1}{2}` is the genus of the curve.
```

Comment:
```suggestion
    \left(\deg(Q)-1\right) / 2` is the genus of the curve.
```
The fraction doesn't look nice in [docs](https://deploy-preview-37424--sagemath.netlify.app/html/en/reference/arithmetic_curves/sage/schemes/hyperelliptic_curves/hypellfrob)

Reply:
OK. Done.